### PR TITLE
OD-735 [Fix] Fixed styles for search result bubble

### PR DESCRIPTION
**Pull request by skrupnyk**
@romanyosyfiv @inna-bieshulia

OD-643 https://weboo.atlassian.net/browse/OD-643

## Description
**Problem:** Styles for LFD search result bubble did not inherit from highlight color.
**Solution:** Added style inheritance from highlight color.

## Screenshots/screencasts
www.awesomescreenshot.com/video/7652864?key=c99c3b484801be85558a91929b30a813

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

File: scss/components/lfd/agenda.scss [hunk 1]
```diff
@@ -644,6 +644,12 @@
         }
       }
 
+      .search-query .current-query-wrapper {
+        background-color: $highlightColor;
```
Comment (tonytlwu) [inline scss/components/lfd/agenda.scss:648]: Use primary button background color

File: scss/components/lfd/agenda.scss [hunk 2]
```diff
@@ -644,6 +644,12 @@
...
+        color: $primaryButtonTextColor;
+        border-color: $highlightColor;
```
Comment (tonytlwu) [inline scss/components/lfd/agenda.scss:650]: Use primary button background color

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: scss/components/lfd/agenda.scss [hunk 3]
```diff
@@ -687,6 +693,12 @@
             color: $primaryButtonTextColorTablet;
           }
 
+          .search-query .current-query-wrapper {
+            background-color: $primaryButtonColorTablet;
+            color: $primaryButtonTextColor;
```
[the remark this answers is not in the document]
Comment (skrupnyk) [reply scss/components/lfd/agenda.scss]: fixed

**Review by tonytlwu — APPROVED** [review]
@romanyosyfiv @inna-bieshulia This is ready for testing

Risk: Medium
Affected: All LFD layouts with search enabled
Verify: Search terms are highlighted using the correct colours

Squash and merge: Use PR title

**Pull request merged**